sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/options/GcsOptions.java

```diff
@@ -129,6 +130,23 @@
 
   void setGcsPerformanceMetrics(Boolean reportPerformanceMetrics);
 
+  /**
+   * Has the effect of performing copy operations with multiple calls to the rewrite API.
+   *
+   * <p>Takes effect only in certain conditions. See {@link
+   * Rewrite#setMaxBytesRewrittenPerCall(java.lang.Long)} for details.
+   *
+   * <p>For testing purposes only.
+   */
+  @Description(
+      "Has the effect of performing copy operations with multiple calls to the rewrite API. "
+          + "For testing purposes only.")
+  @Experimental
+  @Nullable
+  Long getMaxBytesRewrittenPerCall();
```

Comment:
If this is for testing purposes only better to create a new `Options` file thing that extends GcpOptions with this, to avoid it from leaking to the general used options.